geode-core/src/main/java/org/apache/geode/management/internal/operation/OperationHistoryManager.java

```diff
@@ -96,5 +127,17 @@ public String getId() {
     public A getOperation() {
       return operation;
     }
+
+    public Date getOperationStart() {
+      return operationStart;
+    }
+
+    public CompletableFuture<Date> getOperationEnded() {
+      return operationEnded;
+    }
+
+    public void setOperationEnded(Date operationEnded) {
```

Comment:
I find the overloading of "operationEnded" here confusing. Maybe "this.operationEnded" should actually be named "operationEndedFuture" to make clear that it is a container of operationEnded.

Reply:
preferred naming convention seems for Future getXxx seems to be `getFutureXxx`.  I've updated all such CMS methods accordingly.